plugins/uefi-dbx/fu-uefi-dbx-common.c

```diff
@@ -43,17 +175,24 @@ fu_uefi_dbx_signature_list_validate_volume(FuEfiSignatureList *siglist,
 	esp_path = fu_volume_get_mount_point(esp);
 	if (esp_path == NULL)
 		return TRUE;
-	files = fu_path_get_files(esp_path, error);
+	files = fu_uefi_dbx_bootable_efi_path(FU_EFIVAR_GUID_EFI_GLOBAL, error);
```

Comment:
I think you're going to also want to explicitly add to this list _everything_ ending in `*.efi` in the efi removable path (`/efi/boot/*`)